src/Surfnet/ServiceProviderDashboard/Application/CommandHandler/Entity/PublishEntityTestCommandHandler.php

```diff
@@ -75,13 +74,13 @@ public function handle(PublishEntityTestCommand $command)
                     $e->getMessage()
                 )
             );
-            $this->flashBag->add('error', 'entity.edit.error.publish');
+            $this->requestStack->getSession()->getFlashBag()->add('error', 'entity.edit.error.publish');
         }
     }
 
-    private function isNewResourceServer(ManageEntity $entity)
+    private function isNewResourceServer(ManageEntity $entity): bool
     {
-        $isNewEntity = empty($entity->getId());
+        $isNewEntity = $entity->getId() === null || $entity->getId() === '' || $entity->getId() === '0';
```

Comment:
Again, this looks like an incorrect change of behavior

Reply:
It is a rector change, I check iff this is worng or not

Reply:
See my feedback on the review comment above.